frontends/web/src/app.tsx

```diff
@@ -194,7 +196,7 @@ class App extends Component<Props, State> {
                 <Banner msgKey="bitbox02" />
                 <MobileDataWarning />
                 <Aopp />
-                <KeystoreConnectPrompt />
+                {!isManageAccountRoute ? <KeystoreConnectPrompt /> : null}
```

Comment:
Why is this? The keystore prompt can happen anywhere, e.g. when verifying a receive address.